### ISSUE TITLE
Formating brackets []

### ISSUE THREAD

**Issue by cristianvasquez** [sparqling/sparql-formatter#23]
Hi! Thank you for creating this tool!

I'm using it to format some queries, and it beautifies them most of the time. However, I've noticed one issue: brackets ([ and ]) are placed on the same line, making nested structures harder to read.

Would it be possible to add indentation to the structures within brackets for better readability?

```sparql
SELECT *
WHERE {
  ?s a <foo> ;
     <p1> ?p1value ;
     <p2> [ a <something> ;
       <playedBy> [ <name> ?name ;
       <address> [ <hasCountryCode> ?countryUri ] ] ] .
```

While ChatGPT gives me this: 
```sparql
SELECT * WHERE {
  ?s a <foo> ;
     <p1> ?p1value ;
     <p2> [
       a <something> ;
       <playedBy> [
         <name> ?name ;
         <address> [
           <hasCountryCode> ?countryUri
         ]
       ]
     ] 
}
```

**Comment by hchiba1** [comment]
Thanks for the feedback!

I’ve addressed the issue, 
and here is the updated output:
```sparql
SELECT *
WHERE {
  ?s a <foo> ;
     <p1> ?p1value ;
     <p2> [
       a <something> ;
       <playedBy> [
         <name> ?name ;
         <address> [
           <hasCountryCode> ?countryUri
         ]
       ]
     ] .
}
```
The update has been included in the latest release [v0.8.5](https://github.com/sparqling/sparql-formatter/releases/tag/v0.8.5).

**Comment by cristianvasquez** [comment]
Amazing, thanks!
